NEWS.txt

```diff
@@ -3,334 +3,253 @@ Release Notes
- - Add Multinomial Without Replacement
- - conv3d2d support full and half mode (REMOVE?)
- - Add DownsampleFactorMaxGradGrad.grad
- - Allow partial evaluation of compiled function
+ - Added scalar and elemwise ops for modified Bessel function of order 0 and 1 from scipy.special
+ - ``OpFromGraph`` now allows gradient overriding for every input
+ - Added Abstract Ops for batch normalization that use cuDNN when available and pure Theano CPU/GPU alternatives otherwise
+ - Added gradient of solve, tensorinv (CPU), tensorsolve (CPU), searchsorted (CPU), DownsampleFactorMaxGradGrad (CPU)
+ - Added Multinomial Without Replacement
+ - Allowed partial evaluation of compiled function
  - More Rop support
- - Indexing support ellipsis: a[..., 3], a[1,...,3]
- - Added theano.tensor.{tensor5,dtensor5, ...}
+ - Indexing support ellipsis: ``a[..., 3]```, ``a[1,...,3]``
+ - Added ``theano.tensor.{tensor5,dtensor5, ...}``
  - compiledir_format support device
- - Added new Theano flag cmodule.age_thresh_use
+ - Extended  Theano flag ``dnn.enabled`` with new option ``no_check`` to help speed up cuDNN importation
+ - Added New Theano flag ``conv.assert_shape`` to check user-provided shapes at runtime (for debugging)
+ - Added new Theano flag ``cmodule.age_thresh_use``
+ - Added new Theano flag ``cuda.enabled``
+ - Added new Theano flag ``nvcc.cudafe`` to enable faster compilation and import with old CUDA back-end
+ - Added new Theano flag ``print_global_stats`` to print some global statistics (time spent) at the end
+ - Added new Theano flag ``profiling.ignore_first_call``, useful to profile the new gpu back-end
+ - remove ProfileMode (use Theano flag ``profile=True`` instead)
+
 
 Others:
- - Speed up argmax only on gpu (without also needing the max)
- - A few unfrequent bugfix
- - More stack trace in error message
+ - Split op now has C code for CPU and GPU
+ - ``theano-cache list`` now includes compilation times
+ - Speed up argmax only on GPU (without also needing the max)
+ - More stack trace in error messages
  - Speed up cholesky grad
- - log(sum(exp(...))) now get stability optimized
+ - ``log(sum(exp(...)))`` now get stability optimized
+
 
 Other more detailed changes:
- - Allow more then one output to be an destructive inplace
- - Add flag profiling.ignore_first_call, useful to profile the new gpu back-end
- - Doc/error message fixes/updates
+ - Added Jenkins (gpu tests run on pull requests in addition to daily buildbot)
+ - Removed old benchmark directory and other old files not used anymore
+ - Use of 64-bit indexing in sparse ops to allow matrix with more then 2\ :sup:`31`\ -1 elements
+ - Allowed more then one output to be an destructive inplace
  - More support of negative axis
  - Added the keepdims parameter to the norm function
- - Crash fixes
  - Make scan gradient more deterministic
- - Add support for space in path on Windows
- - remove ProfileMode (use Theano flag profile=True instead)
 
+Commiters since 0.9.0rc4:
```

Comment:
I think this list is made on the current master. Theano 0.9 will be the same version as 0.9.0rc4. Not again the master. We have stuff in the master that should not go in 0.9.

I think this will also request some updates of the NEWS.txt file.